src/views/tables/Orderbook.tsx

```diff
@@ -303,7 +303,11 @@ export const Orderbook = ({
   const onRowAction = useCallback(
     (key: Key, row: RowData) => {
       if (currentInput === 'trade' && key !== 'spread' && row?.price) {
-        dispatch(setTradeFormInputs({ limitPriceInput: row.price }));
+        dispatch(
+          setTradeFormInputs({
+            limitPriceInput: MustBigNumber(row.price).toFixed(tickSizeDecimals ?? 2),
```

Comment:
possible to avoid loose magic numbers i.e. `2`?

Reply:
sad its a nullable field, will replace USD_DECIMALS for now!